build/NuSpecs/build-nupkg.ps1

```diff
@@ -108,20 +108,20 @@ function ConfigureNugetCommandLine {
 
 ConfigureNugetCommandLine("Microsoft.WindowsAppSDK")
 
-Copy-IntoNewDirectory -IfExists $BuildOutput\$BuildFlavor\x86\Microsoft.WindowsAppSDK\Microsoft.WindowsAppSDK.dll "$runtimesDir\win10-x86\native"
-Copy-IntoNewDirectory -IfExists $BuildOutput\$BuildFlavor\x86\Microsoft.WindowsAppSDK\Microsoft.WindowsAppSDK.pri "$runtimesDir\win10-x86\native"
-Copy-IntoNewDirectory -IfExists $BuildOutput\$BuildFlavor\x64\Microsoft.WindowsAppSDK\Microsoft.WindowsAppSDK.dll "$runtimesDir\win10-x64\native"
-Copy-IntoNewDirectory -IfExists $BuildOutput\$BuildFlavor\x64\Microsoft.WindowsAppSDK\Microsoft.WindowsAppSDK.pri "$runtimesDir\win10-x64\native"
-Copy-IntoNewDirectory -IfExists $BuildOutput\$BuildFlavor\arm64\Microsoft.WindowsAppSDK\Microsoft.WindowsAppSDK.dll "$runtimesDir\win10-arm64\native"
-Copy-IntoNewDirectory -IfExists $BuildOutput\$BuildFlavor\arm64\Microsoft.WindowsAppSDK\Microsoft.WindowsAppSDK.pri "$runtimesDir\win10-arm64\native"
+Copy-IntoNewDirectory -IfExists $BuildOutput\$BuildFlavor\x86\Microsoft.WindowsAppRuntime\Microsoft.WindowsAppRuntime.dll "$runtimesDir\win10-x86\native"
+Copy-IntoNewDirectory -IfExists $BuildOutput\$BuildFlavor\x86\Microsoft.WindowsAppRuntime\Microsoft.WindowsAppSDK.pri "$runtimesDir\win10-x86\native"
```

Comment:
Do these `.pri` files need renaming, @huichen123  or @axelandrejs ?

Reply:
That's the open question in the PR's description comment block

Reply:
i don't see where it goes into the final packages, and would assume it's just useless build artifact. If the project is a class library, the PRI filename would be the project name. So you want to rename here too. Otherwise this copy becomes even less meaningful.

Reply:
the project appears to be a CppWinRT one. it should not build PRI at all. Please check if that's the case, and if so remove the PRI copy.